frontend/src/shared/AxiosLineTemp.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-row class="justify-center mb-2">
```

Comment:
I would mve this and all  testing code stuff under separate folder test.
As sibling to src I guess? I am not sure myself what it usually done - but can look at our workk project tomorrow..